specifyweb/frontend/js_src/lib/components/Forms/Save.tsx

```diff
@@ -214,27 +221,82 @@ export function SaveButton<SCHEMA extends AnySchema = AnySchema>({
     </ButtonComponent>
   );
 
+  const [carryForwardAmount, setCarryForwardAmount] = React.useState<number>(1);
+
   return (
     <>
       {typeof handleAdd === 'function' && canCreate ? (
         <>
-          {showClone &&
-            copyButton(
-              formsText.clone(),
-              formsText.cloneDescription(),
-              async () => resource.clone(true)
-            )}
+          {resource.specifyTable.name === 'CollectionObject' &&
+          (isInRecordSet === false || isInRecordSet === undefined) &&
+          isSaveDisabled &&
+          showCarry &&
+          showBulkCarry ? (
+            <Label.Inline>
```

Comment:
this Label does not have any actual label text?
if you can't put label text for UI reasons, then remove Label and add placeholder and aria-label attributes